src/quill.mention.js

```diff
@@ -215,9 +232,17 @@ class Mention {
     if (data && data.length > 0) {
       this.values = data;
       this.mentionList.innerHTML = '';
+
       for (let i = 0; i < data.length; i += 1) {
+        const { query } = data[i];
+        let className;
+        if (query === null || query === undefined) {
```

Comment:
Yeah, probably we should look after `className` in `data`. If it exists, we use that. Otherwise we are using `ql-mention-list-item`?

Reply:
But that would make `className` a reserved field, wouldn't it? 

How about this: We could make `className` part of `options` s.t.

```typescript
this.options = {
  // ..
  listItemClass: 'ql-mention-list-item'
  // ..
}
```

This would make it a lot more flexible. E.g. I could decide I want to render each item very specific to in my configuration I set `listItemClass = null` and let my own implementation of `renderItem` decide how each `<li>` gets rendered.

Reply:
Sound good to me. Maybe we should do the same with all className?

https://github.com/afconsult/quill-mention/blob/master/src/quill.mention.js#L50
https://github.com/afconsult/quill-mention/blob/master/src/quill.mention.js#L59

Reply:
@MadSpindel We could do that. Do you want me to implement this? 

In this case `this.option` would expose:

```
this.options = {
  // ..
  listItemClass = 'ql-mention-list-item',
  mentionContainerClass = 'ql-mention-list-container',
  mentionListClass = 'ql-mention-list',
  // ..
}
```

Reply:
Yes, that will be great 👍